Resources/Public/JavaScript/LeafletBackend.js

```diff
@@ -0,0 +1,145 @@
+define(['jquery'], function ($) {
+    'use strict';
+
+    var LeafBE = {
+        $element: null,
+        $gLatitude: null,
+        $gLongitude: null,
+        $latitude: null,
+        $longitude: null,
+        $fieldLat: null,
+        $fieldLon: null,
+        $fieldLatActive: null,
+        $geoCodeUrl: null,
+        $geoCodeUrlShort: null,
+        $tilesUrl: null,
+        $tilesCopy: null,
+        $zoomLevel: 13,
+        $marker: null,
+        $map: null
+    };
+
+    $(function () {
+        // basic variable initalisation
+        LeafBE.$element = $('#location-map-container-a');
+        LeafBE.$latitude = LeafBE.$element.attr('data-lat');
+        LeafBE.$longitude = LeafBE.$element.attr('data-lon');
+        LeafBE.$gLatitude = LeafBE.$element.attr('data-glat');
+        LeafBE.$gLongitude = LeafBE.$element.attr('data-glon');
+        LeafBE.$tilesUrl = LeafBE.$element.attr('data-tiles');
+        LeafBE.$tilesCopy = LeafBE.$element.attr('data-copy');
+        LeafBE.$geoCodeUrl = LeafBE.$element.attr('data-geocodeurl');
+        LeafBE.$geoCodeUrlShort = LeafBE.$element.attr('data-geocodeurlshort');
+        LeafBE.$fieldLat = LeafBE.$element.attr('data-namelat');
+        LeafBE.$fieldLon = LeafBE.$element.attr('data-namelon');
+        LeafBE.$fieldLatActive = LeafBE.$element.attr('data-namelat-active');
+
+        // add the container to display the map as a nice overlay
+        $('body').append(
+            '<div id="t3js-location-map-wrap">' +
+            '<div class="t3js-location-map-title">' +
+            '<div class="btn-group"><a href="#" class="btn btn-default" title="Close" id="t3js-ttaddress-close-map">' +
+            '<img src="/typo3/sysext/core/Resources/Public/Icons/T3Icons/actions/actions-close.svg" width="16" height="16"></a>' +
```

Comment:
don't hardcode that, won't also work in sub directory

Reply:
Done.